src/pages/angels.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState, useEffect } from "react"
+import BrandLayout from "../components/BrandLayout"
+import SEO from "../components/seo"
+import { Button } from "@material-ui/core"
+
+const AngelsPage = () => {
+  const stripe = window.Stripe("pk_test_XiirziufElakjoOpyuyCrPfo")
+  const [sku, setSku] = useState("sku_G7M4igcIzjEhU2")
```

Comment:
You do not need this, as you never use setSku. It was probably used to do something with dropdown in the tutorial you watched.

Reply:
You mean I don't need 
`const [sku, setSku] = useState("sku_G7M4igcIzjEhU2")`